ccloudvm/kata-docker-xenial.yaml

```diff
@@ -30,8 +30,9 @@ write_files:
 runcmd:
 
  - {{beginTask . "Install kata"}}
- - sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_$(lsb_release -rs)/ /' >> /etc/apt/sources.list.d/kata-containers.list"
- - {{proxyVars .}} curl -sL http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+ - ARCH=$(arch)
```

Comment:
Nit: Safe for Ubuntu but I wonder if we want to use `uname -m` for general consistency now (see for example https://github.com/kata-containers/osbuilder/pull/227).